app/services/hyku_addons/update_account_frontend_url.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+require 'uri'
+
+# Update the frontend_url of an account.
+class HykuAddons::UpdateAccountFrontendUrl
+  attr_reader :account
+  attr_reader :frontend_url
+
+  ##
+  # @param account [Account]
+  # @param frontend_url [String] Must be a valid URI
+  def initialize(account, frontend_url)
+    @account = account
+    @frontend_url = frontend_url
+  end
+
+  ##
+  # @return [Boolean] true if validations pass and was able to update the account
+  def save
```

Comment:
What do you think about renaming this to `run` or `perform` or something like that?  I feel like `save` should be reserved for persistable objects like ActiveRecord models.  I'm open to anything though and don't want to block this over the method naming.

Reply:
I think it's a good idea. I just followed an example Service that I found inside the internal test app and used 'save' to keep a bit of consistency and it is, in the end, saving some changes on the Account model, but it feels like 'perform' is a much better name for what Service objects do

Reply:
Interesting.  I didn't know any service classes in Hyku were like that.  The examples I've seen are more like https://github.com/samvera/hydra-works/blob/master/lib/hydra/works/services/upload_file_to_file_set.rb